src/client/App.jsx

```diff
@@ -34,16 +34,55 @@ function App() {
    * */
 
   const handleRegister = async ({ username, password }) => {
-
+    const createdRegister = await fetch(`${apiUrl}/user/register`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ username, password }),
+    });
+    console.log(createdRegister);
   };
 
   const handleLogin = async ({ username, password }) => {
-
+    const verifyLogin = await fetch(`${apiUrl}/user/login`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ username, password }),
+    });
+    const logInToken = await verifyLogin.json();
+    localStorage.setItem("token", JSON.stringify(logInToken));
   };
 
   const handleCreateMovie = async ({ title, description, runtimeMins }) => {
+    try {
+        
+        const token = JSON.parse(localStorage.getItem("token"));
```

Comment:
would be good if you checked this value after parsing it. What if the token is undefined in local storage?